components/quiz.js

```diff
@@ -57,24 +59,41 @@ export default function Quiz (props) {
 						{answer['answer']}
 					</div>
 				</div>
-
 				<div>
 					<Row className="p-3">
 						<Col xs={4}>
-							<Button className='actionButton' variant={yes} onClick={e => handleClick(e, 'yes')}>Yes</Button>
+							<Button className='actionButton' variant={yes} disabled={next} onClick={e => handleClick(e, 'yes')}>Yes</Button>
 						</Col>
 						<Col xs={4}>
-							<Button className='actionButton' variant={no} onClick={e => handleClick(e, 'no')}>No</Button>
+							<Button className='actionButton' variant={no} disabled={next} onClick={e => handleClick(e, 'no')}>No</Button>
 						</Col>
 						<Col xs={4}>
-							<Button className='actionButton' variant={maybe} onClick={e => handleClick(e, 'maybe')}>Unsure</Button>
+							<Button className='actionButton' variant={maybe} disabled={next} onClick={e => handleClick(e, 'maybe')}>Unsure</Button>
 						</Col>
 					</Row>
 				</div>
 			</main>
 
 			<footer className="mt-auto next-section">
-				<Button variant="primary" onClick={handleNext} disabled={!next}><span>NEXT<img className="white" src="/white.svg"/></span></Button>{' '}
+				{next
+					? <div>
+						<ProgressBar 
+							label="Yes" 
+							color={result.answer == 'yes' ? '#0068ea' : '#808080'}
```

Comment:
I would suggest to replace these hardcoded values with constants. Those constants should be defined at the top of the file, or better yet, create another file, and have one single source of truth for constants (importing it wherever these values are used). Such a file does not exist yet because I don't recall hardcoding values like this in the code, but maybe I did so inadvertently (and will update them if I find them).

This will make this code more easily updatable/customizable in the future, where we will only have to change these values once; as currently the first value is both used in this file and `components/progressBar.js`